source/_posts/2017-01-21-home-assistant-governance.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Home Assistant Governance"
+description: "Protecting Home Assistant for all, now and into the future"
+date: 2017-01-21 15:05:00 -0800
+date_formatted: "January 21, 2017"
+author: Paulus Schoutsen
+author_twitter: balloob
+comments: true
+categories: Organisation
+---
+
+The Home Assistant project and community has seen enormous growth in the last three years. Hundreds, if not thousands, of volunteers have and are working tirelessly every day on giving you all the awesomeness that you see today. That's not just the code but also all the related projects, documentation, examples, release blog posts and help in our forums and chat. And this is something that we want to keep safe and functional, even as we grow.
+
+Starting today we are announcing a few initiatives to help protect our users, contributors and community members.
+
+## Code of Conduct
+
+More and more people are getting to know and love Home Assistant. Our community keeps growing and growing. Our community consists of people from all over the world with different backgrounds and we want Home Assistant to be a place where everyone can feel at home. To help with this we're introducing a Code of Conduct. The Code of Conduct describes what type of behavior is unacceptable, how we will enforce it and where to report incidents.
```

Comment:
`More and more` `growing and growing`

Using adjectives, adverbs, or simply more descriptive words would really improve the tone of these sentences. Don't take it from me, though, take it from Robin Williams.

https://www.youtube.com/watch?v=2EdWgsTUhmI